lib/css/modules/_dashboard.scss

```diff
@@ -374,3 +374,21 @@ ul.widgetStateButtons {
 #userTaggerTable tr:hover .deleteIcon {
 	color: #369;
 }
+
+.res-r-dashboard {
```

Comment:
This class isn't guaranteed to be present since the stylesheet module can be turned off; we should maybe put it in /r/Dashboard CSS